application/controllers/catalog/Page.php

```diff
@@ -131,9 +131,9 @@ function _read_by($project)
 	{
 		if (strtolower($project->project_type) == 'group') return 'LibriVox Volunteers';
 
-		$readers = $this->project_model->get_project_readers($project->id);
+		$readerstub = $this->project_model->get_solo_reader($project->id);
 
-		if ($readers) return '<a href="' . base_url() . 'reader/' . $readers[0]['reader_id'] . '">' . $readers[0]['display_name'] . '</a>';
+		if (isset($readerstub)) return '<a href="' . base_url() . 'reader/' . $readerstub['reader_id'] . '">' . $readerstub['display_name'] . '</a>';
```

Comment:
The fact that we return inline HTML from the controller is janky, but you didn't write that, and it's out of scope in this PR to fix.